### PR TITLE
add newsletter examples

### PR DESCRIPTION
**Pull request by diodon**
This is the code used to create the plot for the second newsletter

### CODE REVIEW

File: newsletter-examples/NOAA-CRW-DHW.ipynb [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    "## Extraction of a time series of Degree Heating Weeks (DHW) from the NOAA CRW collection\n",
+    "\n",
+    "*This is the notebook used to create the timeseries plot showcased in the DMS Newsletter #2*\n",
+    "\n",
+    "If you want to try this notebook using NECTAR's JupyterHub service just click here (you need to have an academic email registered with AAF): [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ausseabed/dms-newsletter-2/HEAD?filepath=notebooks%2FDHW_timeseries.ipynb)\n",
+    "\n",
+    "What you need to know:\n",
+    "- the location of the Great Barrier Reef you want to assess. It must be a single point (lat, lon)\n",
+    "- the time period you want to assess. It must be a range of years (e.g. 2010-2019)\n",
+    "- the name of the output file you want to create. It must be a string (e.g. 'my_output_file.csv')\n",
```
Comment (LeoLee-Xiaohu) [inline newsletter-examples/NOAA-CRW-DHW.ipynb:18]: What's the output CSV used for? There are no codes in this notebook to generate CSV.

File: newsletter-examples/NOAA-CRW-DHW.ipynb [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "outputs": [],
+   "source": [
+    "import xarray as xr\n",
+    "import pandas as pd\n",
```
Comment (LeoLee-Xiaohu) [inline newsletter-examples/NOAA-CRW-DHW.ipynb]: There is no use for `pandas`. It's better to remove `pandas`. If we want to save `zarr ` as `csv`, we could use `xarray ` directly to save it as a csv, something like subzarr.to_dataframe().to_csv("my_output_file.csv").

File: newsletter-examples/NOAA-CRW-DHW.ipynb [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "latitude = -18.5\n",
+    "longitude = 147.5\n",
+    "time_range = ['1995', '2023']\n",
+    "output_file = 'my_output_file.csv'\n",
```
Comment (LeoLee-Xiaohu) [inline newsletter-examples/NOAA-CRW-DHW.ipynb:70]: There is no use for output_file as well.

**Comment by LeoLee-Xiaohu** [comment]
Looks good! Some modifications were made to explain the notebook better. For you refer: https://github.com/aodn/rimrep-examples/blob/766cf7016b4cfc037bd027c9f42199c967b06117/newsletter-examples/NOAA-CRW-DHW.ipynb

File: newsletter-examples/NOAA-CRW-DHW.ipynb (hunk 1 above)
Comment (diodon) [reply newsletter-examples/NOAA-CRW-DHW.ipynb:18]: keep as it will be used at the end

File: newsletter-examples/NOAA-CRW-DHW.ipynb (hunk 2 above)
Comment (diodon) [reply newsletter-examples/NOAA-CRW-DHW.ipynb]: agree

File: newsletter-examples/NOAA-CRW-DHW.ipynb (hunk 3 above)
Comment (diodon) [reply newsletter-examples/NOAA-CRW-DHW.ipynb:70]: it will be used at the end

**Comment by diodon** [comment]
All comments addressed, Please have a look again @LeoLee-Xiaohu

**Review by LeoLee-Xiaohu — APPROVED** [review]
LGTM!

**Pull request merged**